src/govuk/components/radios/radios.js

```diff
@@ -55,8 +55,12 @@ Radios.prototype.handleClick = function (event) {
   //
   // We also only want radios which have aria-controls, as they support conditional reveals.
   var $allInputs = document.querySelectorAll('input[type="radio"][aria-controls]')
+  var $clickedInput = event.target
   nodeListForEach($allInputs, function ($input) {
-    this.setAttributes($input)
+    // In radios, only radios with the same name will affect each other.
```

Comment:
Can we have a test for this? :thinking:

Reply:
Because all this will do is sync the conditional reveals with the checked state of their inputs, we'd have to do something really weird, like having another set of radios with a conditional reveal that's 'out of sync' with its radios, and asserting that it doesn't sync them back up again.

It's such a synthetic scenario I'm not sure it makes much sense as a test.